Tests/SentryTests/Transaction/SentryTracer+Test.h

```diff
@@ -10,4 +10,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+void testing_setMeasurementWithNilName(SentryTracer *tracer, NSNumber *value);
```

Comment:
`l`: Should we mark this explicitly as `_Nullable`?

Reply:
The tracer and value must not be nil. The implementation passes `nil` to the setMeasurement. So not it doesn't make sense.